certbot/storage.py

```diff
@@ -93,6 +94,13 @@ def write_renewal_config(o_filename, n_filename, archive_dir, target, relevant_d
 
     """
     config = configobj.ConfigObj(o_filename)
+    if not "version" in config \
```

Comment:
nit: You could probably simplify this logic a bit by using `.get()` on `config` and providing an optional parameter of either `None` or a version number lower than 0.14.0.

This logic works as is though and I'll leave it up to you whether you make this change.